src-docs/src/views/health/health_example.js

```diff
@@ -27,10 +27,11 @@ export const HealthExample = {
       text: (
         <p>
           The <strong>EuiHealth</strong> component should be used when showing
-          comparitive health of listed objects (like servers, nodes,
-          indexes..etc). Because icons are vague and bulky and color alone does
-          not work, we think color plus text provides a recognizable,
-          lightweight combo that works in most situations.
+          comparitive health of listed objects (like servers, HTTP response
+          status codes(as per convenience), nodes, indexes..etc). Because icons
+          are vague and bulky and color alone does not work, we think color plus
```

Comment:
Nit from reading our old copy:
```suggestion
          are vague and bulky and color alone does not work, color plus
```